interim-jan-2021/intro.md

```diff
@@ -0,0 +1,42 @@
+# Emerging Use Cases for Encrypted DNS
+## Potential next steps for ADD
+
+For the IETF ADD interim, January 2021
+
+Authors:
+* Chris Box
+* Tommy Jensen
+* Tiru Reddy
+* Jim Reid
+* Ben Schwartz
+
+----
+# Overview
+
+* draft-box-add-requirements describes what we need to upgrade from unencrypted to encrypted.
+* DEER provides a mechanism for this.
+* What's next for ADD?
+* In these slides: three scenarios beyond simple upgrade, with proposed requirements
+----
+# Three scenarios
+
+1. DNS configuration on explicitly trusted networks
+2. Resolver behavior self-description
+3. Publishing and using directories of encrypted resolvers
+----
+# Goals
+
+* Solicit feedback on scope and requirements
+* Gauge interest in possible next steps for the WG
+* Proceed with proper requirements drafts where there is interest
+* Support compliance and compatibility with other IETF standards
+  - e.g. [Unknown RR types](https://tools.ietf.org/html/rfc3597), [DNSSEC](https://tools.ietf.org/html/rfc4035), [Extended DNS Errors](https://tools.ietf.org/html/draft-ietf-dnsop-extended-error)
```

Comment:
A compliant DNS proxy must not drop unknown RR types, see https://tools.ietf.org/html/bcp152#section-4.3. 
Why would a non-complaint DNS proxy indicate non-support of the the BCP ?
The client would anyway know whether the server supports DNSSEC or not (what is the need for a explicit signal from the server) ?

Reply:
> A compliant DNS proxy must not drop unknown RR types, see https://tools.ietf.org/html/bcp152#section-4.3.

I agree.

> Why would a non-complaint DNS proxy indicate non-support of the the BCP ?

This text is intended to clarify that we are trying to push in the other direction, toward better standards compliance.  We want to make it easy for the proxy to comply with the BCP, and avoid encouraging non-compliance.

> The client would anyway know whether the server supports DNSSEC or not (what is the need for a explicit signal from the server) ?

This isn't proposing an explicit signal; it's saying that "our goal is to encourage compliance and compatibility with DNSSEC".  For example, in some proposals, the recursive resolver can override the authoritative resolver and produce different answers for a zone (e.g. split horizon).  This text is about trying to find a way to remain compatible with DNSSEC in scenarios like that.

Reply:
I get it. It is one way to force DNS proxies to be compatible with IETF recommendations :)